tests/main/dbus-activation-session/task.yaml

```diff
@@ -0,0 +1,49 @@
+summary: D-Bus session services support activation via systemd user session
+
+systems:
+    # Ubuntu Core does not contain the D-Bus user session systemd units
+    - -ubuntu-core-*
+    # Ubuntu 14.04 does not provide user@.service
+    - -ubuntu-14.04-*
+    # CentOS/Amazon Linux give error "Unit user@0.service not loaded."
+    - -amazon-linux-2-*
+    - -centos-7-*
+
+prepare: |
+    snap set system experimental.user-daemons=true
+    snap set system experimental.dbus-activation=true
+    #shellcheck source=tests/lib/pkgdb.sh
+    . "$TESTSLIB"/pkgdb.sh
+    distro_install_package dbus-user-session
+    session-tool -u test --prepare
+
+restore: |
+    session-tool -u test --restore
+    snap unset system experimental.dbus-activation
+    snap unset system experimental.user-daemons
+
+execute: |
+    #shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB"/snaps.sh
+
+    echo "Install a snap containing an activatable D-Bus session service"
+    snap install --edge test-snapd-dbus-service
+
+    echo "A service activation file has been created"
+    [ -f /var/lib/snapd/dbus/services/io.snapcraft.SnapDbusService.service ]
+
+    echo "The service is not initially running"
+    not session-tool -u test systemctl --user is-active snap.test-snapd-dbus-service.session.service
```

Comment:
Could you please update the spread test to the new command syntax:

`session-tool --prepare` => `tests.session prepare`
`session-tool <cmd>` => `tests.session exec <cmd>`
`session-tool --restore` => `tests.session restore`